src/software/ai/hl/stp/tactic/crease_defender/crease_defender_fsm.h

```diff
@@ -107,19 +109,9 @@ struct CreaseDefenderFSM
     }
 
    private:
-    /** Max ratio between distances (crease and ball) / (crease and nearest enemy) for
-     * crease to chase ball. Scale from (0, 1)
-     * |------------------------------------------|
-     * | Crease | <-----------------------> Enemy |
-     * |        | <----> Ball                 |   |
-     * |       ()         x                  ()   |
-     * |------------------------------------------|
-     */
-    static constexpr double MAX_GET_BALL_RATIO_THRESHOLD = 0.3;
-    // Max distance that the crease will try and get possession of a ball
-    static constexpr double MAX_GET_BALL_RADIUS_M = 1;
-    // Max speed of ball that crease will try and get possession
-    static constexpr double MAX_BALL_SPEED_TO_GET_MS = 0.5;
+    static constexpr double DETECT_THREAT_AHEAD_SHAPE_LENGTH_M = 1;
+    static constexpr double DETECT_THREAT_AHEAD_SHAPE_RADIUS_M = 0.1;
+    const Vector GOALPOST_OFFSET                               = Vector(0, 0.5);
```

Comment:
nit: You should add `GOALPOST_OFFSET` to the dynamic params. I can see us wanting to tune this in a game if we're chipping a lot. For the dynamic param, it also could just be a single double defining the distance in the y-axis.